frontend/.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
What does this `.env` file include? Is it the file containing your keys?

Reply:
I've hidden the link to our API server there. Will send over the files: there are three .env files: one for frontend, one for api server, one for telegram bot